tests/screenshot/gen_screenshots.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2019 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Node.js script to generate screenshots in Chrome, via webdriver.
+ */
+var webdriverio = require('webdriverio');
+var fs = require('fs');
+
+module.exports = genScreenshots;
+
+/**
+ * Opens two different webdriverio browsers.  One uses the hosted version of
+ * blockly_compressed.js; the other uses the local blockly_uncompressed.js.
```

Comment:
Why generate the golden images on the fly instead of checking in a set? I'm not saying it's the wrong approach, but it's worth commenting on.

Reply:
Agreed.

1) I needed to set up this playground and script anyway in order to generate the golden images, whether I'm checking them in or generating on the fly
2) I anticipate that I'll tweak the parameters in the playgrounds for a while (zoom level? total size in pixels?) and don't want to check in intermediate stuff

I *am* trying to make sure that switching between generating on the fly and checking in a set of golden images is very easy, so that we can check them in later.